src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -353,6 +353,15 @@ private BoundExpression BindNamedAttributeArgument(AttributeArgumentSyntax named
 
             ReportDiagnosticsIfObsolete(diagnostics, namedArgumentNameSymbol, namedArgument, hasBaseReceiver: false);
 
+            if (namedArgumentNameSymbol.Kind == SymbolKind.Property)
+            {
+                var propertySymbol = (PropertySymbol)namedArgumentNameSymbol;
+                if (propertySymbol.SetMethod != null)
```

Comment:
>propertySymbol [](start = 20, length = 14)

If the property is overridden, which property do we get here the one from the base, or the one from the derived type? Derived type doesn't have to override all accessors. If we are dealing with an overriding property that overrides only getter, we will fail the null check and will not check the setter for obsoleteness. Please add a test for a scenario like that.

Reply:
Thanks for spotting that. Fixed.